src/c20_server/regulations_api_errors.py

```diff
@@ -0,0 +1,15 @@
+"""
+a set of custom errors specific to the regulations.gov api
+"""
+
+
+class IncorrectApiKey(Exception):
+    pass
+
+
+class ThousandCalls(Exception):
+    pass
+
+
+class BadID(Exception):
```

Comment:
How about 'BadDocumentIDException`?  

or is this used for other IDs?

Reply:
This is an exception thrown for both the documents and the dockets, I was thinking something more along the lines of BadDocIDException if we do not want to leave it as BadIDException.